AGENTS.md

```
@@ -1,30 +1,59 @@
-    - ユーザーのやりたいことが1Spec=1PRとして過剰・過大であるかを判断。そうである場合、適切な要件と規模に落とし込む(skill: sdd-slice-wish)
-    - ユーザーからヒアリングした内容を元に、Specのドラフトを作成する(skill: sdd-init)
-    - ドラフトの内容に合意できたら、TDDのための充分なテストケースをSpecに網羅する(skill: sdd-test-cases)
-1. Red: 失敗するテストを書く(skill: tdd-red)
-1. Green: テストをパスさせる(skill: tdd-green)
-1. Refactor: コードの品質を向上させる(skill: tdd-refactor)
-1. Commit: 進捗を保存する
+# AI-DLC and Spec-Driven Development
+
+Kiro-style Spec Driven Development implementation on AI-DLC (AI Development Life Cycle)
+
+## Project Memory
```

markdownlint-cli2 (0.20.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)